### ISSUE TITLE
Provide option to reboot after Boxstarter script is complete

### ISSUE THREAD

**Issue by schittli** [chocolatey/boxstarter#519]
Good evening

I discussed this subject in discord and was told to create a Feature Request if we really need it. Thanks a lot for this support and this possibility!

## Is Your Feature Request Related To A Problem? Please describe.
The Feature Request is related to this quality workflow:
1. I use BoxStarter to initialize new Notebooks for non-profit organizations and afterward, I send those notebooks far away.
2. Therefore I start BoxStarter, it works for about 50 minutes and installs Windows Updates, too
3. Afterward, as part of the quality assurance, I must reboot the computer and validate that the most important things are ok.

For example, I ensure:
- that Windows actually boots and it doesn't take 40 minutes to finish installing updates. It's great if users can just start working 😄 
- that all Windows updates have actually been installed which BoxStarter cannot always guarantee (and that's fine because Windows updates are still not reliable).
- That the most critical applications, such as OpenVPN and Office, work, are configured correctly, and have the right licenses.

Of course, I can restart the computer manually after the BoxStarter installation. But because there are **very useful** use-cases in which the restart is part of the installation process, it would be great if BoxStarter could do it too.

## Describe The Solution. Why is it needed?

Therefore, 
### it would be great if BoxStarter would offer a `reboot command` which does the following:
- Cleanup all BoxStarter configurations so that, for example, 
- there is no longer an autologin 
- and BoxStarter does not continue with the installation script after login

Put simply:
A `reboot command` that disables BoxStarter and then restarts the computer normally.

By the way, I am responsible for several hundred employees at work and I would like to introduce Chocolatey & BoxStarter there as well. But there, too, it is part of the quality control that we restart the computer after the SW installation is done and check the most important things.

## Additional Context.
N/A

## Related Issues
N/A

Thanks a lot, kind regards,
Thomas

**Comment by mwallner** [comment]
Without getting into all the detail, may I suggest you add 

```
Install-WindowsUpdate -AcceptEula
```

at the very end of your Boxstarter install script(s) ?
I suspect this would solve the problem at hand.

**Comment by pauby** [comment]
There has been no updates to this in some time, so I'm going to go ahead and close this, but we can reopen it if necessary.
